examples/webgl_reverse_depth_buffer.html

```diff
@@ -105,10 +103,20 @@
 			import * as THREE from 'three';
 
 			import Stats from 'three/addons/libs/stats.module.js';
+			import { EffectComposer } from 'three/addons/postprocessing/EffectComposer.js';
+			import { RenderPass } from 'three/addons/postprocessing/RenderPass.js';
+			import { OutputPass } from 'three/addons/postprocessing/OutputPass.js';
 
-			let stats, camera, scene, normalRenderer, logarithmicRenderer, reverseRenderer;
+			let stats, camera, scene;
+			let normalRenderer, logarithmicRenderer, reverseRenderer;
+			let normalComposer, logarithmicComposer, reverseComposer;
 			const meshes = [];
 
+			const renderTarget = new THREE.WebGLRenderTarget();
+			renderTarget.type = THREE.UnsignedByteType;
```

Comment:
Any pointers to what setting the `type` property does? I don't see it documented for `WebGLRenderTarget`.

Reply:
I guess this should be:
```js
renderTarget.texture.type
```
But since the default is already `THREE.UnsignedByteType` we probably can just delete this line.

Reply:
This is a bad habit of mine. I will remove the line #30989.